.travis.yml

```diff
@@ -26,9 +26,11 @@ env:
     global:
         - NUMPY_VERSION=1.10
         - ASTROPY_VERSION=1.1
+        - LIBGFORTRAN_VERSION=1.0
         - CONDA_DEPENDENCIES='Cython click scipy h5py matplotlib pyyaml scikit-image scikit-learn pandas pyregion naima photutils wcsaxes sherpa'
+        - CONDA_DEPENDENCIES_LINUX='$CONDA_DEPENDENCIES libgfortran=$LIBGFORTRAN_VERSION'
```

Comment:
@bsipocz - Is this the best way to handle the case where the conda install instructions are different on Linux and Mac? Or is there a better way?